.github/workflows/continuous-integration-windows.yml

```diff
@@ -20,14 +20,16 @@ jobs:
       id: cuda-toolkit
       with:
         cuda: '12.4.1'
+        method: 'network'
+        sub-packages: '["nvcc","cudart","thrust"]'
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
     - name: configure
       shell: bash
       run: |
         mkdir build
         mkdir c:/project
         cd build
-        cmake -DKokkos_ENABLE_CUDA=ON -DKokkos_ARCH_VOLTA70=ON -DKokkos_ENABLE_TESTS=ON -DKokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE=ON ..
+        cmake -DKokkos_ENABLE_CUDA=ON -DKokkos_ARCH_VOLTA70=ON -DKokkos_ENABLE_TESTS=ON -DKokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE=ON -DCMAKE_CXX_COMPILER="${CUDA_PATH}/bin/nvcc"
```

Comment:
Is that the right syntax for paths on this platform?

Reply:
whoops ... my bad

Reply:
fixed the path to the source in https://github.com/kokkos/kokkos/pull/7835/commits/c0ad4736be4cc34aaf3e727e992cccd26e4f7cc6

Reply:
> Is that the right syntax for paths on this platform?

AFAIK CMake will prefer forward slashes